README.md

```diff
@@ -1,36 +1,30 @@
-# muser-firebase-export ![Java CI with Maven](https://github.com/CUTR-at-USF/muser-firebase-export/workflows/Java%20CI%20with%20Maven/badge.svg)
```

Comment:
Not sure why there are a lot of changes to the README here, including removing the markdown formatting?

README should remain intact, with the only changes being the addition of the new date range command-line parameter.